config/test/config.myopic.yaml

```diff
@@ -40,6 +40,7 @@ sector:
     - air
     - geothermal
     - river_water
+    - lake_water
```

Comment:
The dataset is quite large (750 MB). At the moment we run into storage limits in the CI. Therefore, I think we have to remove this option from the CI configs.